compiler/runtime/OMRCodeCacheManager.cpp

```diff
@@ -1069,14 +1076,16 @@ OMR::CodeCacheManager::carveCodeCacheSpaceFromRepository(size_t segmentSize,
          }
       }
 
-   if (config.verboseCodeCache())
+   if (start)
       {
-      if (start)
+      if (config.verboseCodeCache())
          TR_VerboseLog::writeLineLocked(TR_Vlog_CODECACHE, "carved size=%u range: " POINTER_PRINTF_FORMAT "-" POINTER_PRINTF_FORMAT,
                   codeCacheSizeToAllocate, start, end);
-      else
-         TR_VerboseLog::writeLineLocked(TR_Vlog_FAILURE, "failed to carve size=%lu. Free space = %u",
-                  codeCacheSizeToAllocate, freeSpace);
+      }
+   else
+      {
+      if (config.verbosePerformance())
```

Comment:
Before this change, we would print a failure message for `verbose={codeCache}`. After this change we will print a failure message only for `verbose={compilePerformance}`. I think that the failure message should be printed for either of those two verbose options because it's about code cache and the failure has performance implications.

Reply:
Agree. I have included both verbose option for failure logging now.